classes/classes/BodyParts/BodyPart.as

```diff
@@ -0,0 +1,47 @@
+/**
+ * Created by aimozg on 25.04.2017.
+ */
+package classes.BodyParts {
+import classes.CoC_Settings;
+import classes.Creature;
+import classes.internals.SimpleJsonable;
+
+public class BodyPart extends SimpleJsonable {
+	include "../../../includes/appearanceDefs.as";
```

Comment:
@brrritssocold The Claws has the example of where I'd like to have the enum constants.
And the has-is helpers in Face is what should be used from outside.